README.md

```diff
@@ -48,7 +50,7 @@ Browsers Requirement
 Prerequisite
 ---
 
-- Node.js 4.x or up (test up to 6.7.0)
+- Node.js 6.x or up (test up to 6.7.0)
```

Comment:
In travis, already use latest node 7 version to test.
Please modify this line to node 7 latest version.